src/main/java/mesosphere/marathon/client/model/v2/HealthCheck.java

```diff
@@ -12,7 +12,10 @@
 	private Integer timeoutSeconds;
 	private boolean ignoreHttp1xx;
 	private String path;
-	private String protocol;
+	// protocol is http, https
+    private String protocol;
```

Comment:
nit pick: alignment

Reply:
yeah... my editor complained that tabs were used instead of spacing... I'm inclined to reformat the whole file but then it is hard to see what changed